demisto_sdk/commands/content_graph/strict_objects/asset_modeling_rule_schema.py

```diff
@@ -0,0 +1,14 @@
+from demisto_sdk.commands.content_graph.strict_objects.common import create_model
+from demisto_sdk.commands.content_graph.strict_objects.modeling_rule_schema import (
+    StrictModelingRuleSchema,
+)
+
+# This model/class represents the StrictAssetsModelingRuleSchema.
+# The AssetsModelingRuleSchema and ModelingRuleSchema objects have the same schema,
+# but they are separate for future changes.
+
+
+StrictAssetsModelingRuleSchema = create_model(
```

Comment:
may be simpler to use
```python
class StrictAssetsModelingRuleSchema(StrictModelingRuleSchema):
    ...
```
rather than call `create_model`

Reply:
no problem, reverting it